nearcore/src/config.rs

```diff
@@ -82,7 +82,7 @@ const BLOCK_HEADER_FETCH_HORIZON: BlockHeightDelta = 50;
 const CATCHUP_STEP_PERIOD: u64 = 100;
 
 /// Time between checking to re-request chunks.
-const CHUNK_REQUEST_RETRY_PERIOD: u64 = 400;
+const CHUNK_REQUEST_RETRY_PERIOD: u64 = 800;
```

Comment:
optional: should we also change any settings in
https://github.com/near/nearcore/blob/master/chain/chunks/src/lib.rs#L134 ?

(in theory - we can leave them as is)

Reply:
@mzhangmzz are we confident that this won't degrade the performance, i.e, the time it takes to reconstruct a chunk? Maybe we should test this PR and see how well it performs